docs/sql/map.md

```diff
@@ -21,6 +21,15 @@ maps's values are nullable `INT` values.
 Map literals have the syntax `MAP[`key `,` value ( `,` key `,` value )* `]`.
 Here is an example: `SELECT MAP['hi',2]`.
 
+## Comparison Operations on Maps
+
+Comparison operations (`=`, `<>`, `!=`, `>`, `<`, `>=`, `<=`) can be applied to maps. First, the keys are sorted in lexicographical order. The comparison occurs on the keys first. If the keys are equal, their corresponding values are compared next.
```

Comment:
I am guessing that if two keys are equal, then their values are compared.
If the values are equal, then next key is compared.
I don't think this is what you wrote.